packages/pds/src/image/uri.ts

```diff
@@ -59,8 +56,26 @@ export class ImageUriBuilder {
         width: 250,
         min: true,
       })
+    } else if (id === 'feed_fullsize') {
+      return this.getSignedUri({
+        cid: typeof cid === 'string' ? CID.parse(cid) : cid,
+        format: 'jpeg',
+        fit: 'cover',
+        height: 500,
+        width: 500,
+        min: true,
+      })
+    } else if (id === 'feed_thumbnail') {
+      return this.getSignedUri({
+        cid: typeof cid === 'string' ? CID.parse(cid) : cid,
+        format: 'jpeg',
+        fit: 'cover',
+        height: 250,
+        width: 250,
+        min: true,
+      })
     }
-    return undefined
+    throw new Error(`Unrecognized requested common uri type: ${id}`)
```

Comment:
If exhaustive checks give you warm fuzzies, here's a little TS pattern for it: https://github.com/bluesky-social/atproto/blob/eb6f295c52ca7b484ef771a8aea0e90bcebf2fae/packages/pds/src/api/app/bsky/feed/getTimeline.ts#L81-L84

Reply:
oh hey that's how you get the compiler to pipe down 🙇